scripts/lib/devtool/ide.py

```diff
@@ -1074,6 +1077,10 @@ def setup_ide(self, args, image, gdb_cross):
         if args.ide == 'none' and self.build_tool == BuildTool.CMAKE:
             self.none_launch(image, gdb_cross)
 
+        if(image.gdbserver_missing):
+            logger.warning(
+                "gdbserver not installed in image. Remote debugging will not be available")
```

Comment:
So far this plugin is used for formatting : https://github.com/microsoft/vscode-autopep8. This patch introduces 2 little issues.

Reply:
Thanks for the hint, I ran the extension and fixed formatting.